c++/src/kj/mutex-test.c++

```diff
@@ -73,15 +73,19 @@ TEST(Mutex, MutexGuarded) {
 #if KJ_USE_FUTEX
     auto timeout = MILLISECONDS * 50;
 
-    auto startTime = systemPreciseMonotonicClock().now();
-    EXPECT_TRUE(value.lockExclusiveWithTimeout(timeout) == kj::none);
-    auto duration = startTime - systemPreciseMonotonicClock().now();
-    EXPECT_TRUE(duration < timeout);
```

Comment:
I think the intent here was actually to verify that we waited for at least the expected amount of time before giving up on the lock. So, with a correctly-calculated `duration`, we actually want this to be:

```
EXPECT_TRUE(duration >= timeout);
```

This expectation should be safe against flakes, since if the test runs overly slowly, it will only make `duration` increase further.

Reply:
(That is to say, I think we should bring the expectation back, corrected.)

Reply:
Fixed in https://github.com/capnproto/capnproto/compare/e54256c7aa760d638f43ac612fde44027b5dfea2..3f037ce9b166f180a789bbc8cfda52a1ff268408